core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -262,14 +264,20 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
       // This can happen if there's
       // - no azure config
       // - no billing profile or the billing profile's cloud platform is GCP
-      workspace <- workspaceOpt.flatten
+      (workspace, cloudPlatform) <- workspaceOpt.flatten
         .map(Future.successful)
         .getOrElse(
           traceFutureWithParent("createWorkspace", parentContext) { s =>
-            workspaceService.createWorkspace(workspaceRequest, s)
+            workspaceService
+              .createWorkspace(workspaceRequest, s)
+              .map(workspace => (workspace, WorkspaceCloudPlatform.Gcp))
           }
         )
-    } yield workspace
+    } yield WorkspaceDetails.fromWorkspaceAndOptions(workspace,
+                                                     Some(workspaceRequest.authorizationDomain.getOrElse(Set.empty)),
+                                                     useAttributes = true,
```

Comment:
The `WorkspaceDetails` constructor we used to call set `useAttributes` to true.

`fromWorkspaceAndOptions(workspace, Option(authorizationDomain), useAttributes = true)`